WooCommerce/Classes/ViewModels/Booking Details/HeaderContent.swift

```diff
@@ -9,10 +9,16 @@ extension BookingDetailsViewModel {
         let status: [Status]
 
         init(_ booking: Booking) {
-            bookingDate = booking.startDate.formatted(date: .numeric, time: .omitted)
+            bookingDate = booking.startDate.formatted(.dateTime
+                .month(.twoDigits)
+                .day(.twoDigits)
+                .year(.defaultDigits)
+                .hour(.twoDigits(amPM: .abbreviated))
+                .minute(.twoDigits)
+            )
```

Comment:
❓ Is it necessary to keep the strict format? I'd prefer keeping the previous format to display the format matching the device locale. E.g: if my phone is US locale I have 4:00pm  but if I switch to Vietnam I get 16:00. Matching device locale is more user friendly IMO.